MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m

```diff
@@ -33,12 +33,15 @@
 
 @implementation MSDistribute
 
+@synthesize channelConfiguration = _channelConfiguration;
+
 #pragma mark - Service initialization
 
 - (instancetype)init {
   if ((self = [super init])) {
     _apiUrl = kMSDefaultApiUrl;
     _installUrl = kMSDefaultInstallUrl;
+    _channelConfiguration = [[MSChannelConfiguration alloc] initDefaultChannelWithGroupID:[self groupID]];
```

Comment:
Actually we don't have any requirements/specs for that. For now, we don't send any logs so change that to default value.